### DISCUSSION TITLE
Internet access to models

### DISCUSSION THREAD

**Discussion by sassanix** [open-webui/open-webui#889] [Ideas]
Is it possible to give internet access to the different models?

I'm trying to figure it out, but if you know how, that would be great.

**Comment by justinh-rahb** [comment]
First type **#** followed by a URL and then click on it when it pops up above the text box:
<img width="760" alt="Screenshot 2024-02-19 at 8 34 23 PM" src="https://github.com/open-webui/open-webui/assets/52832301/233a14e6-a39e-4faf-bbd5-681fdcace901">

Once you've done that, the web page is now indexed into RAG and you can ask a question about it:
<img width="760" alt="Screenshot 2024-02-19 at 8 34 29 PM" src="https://github.com/open-webui/open-webui/assets/52832301/b798d6d8-5626-42eb-9de4-4715a62d3d67">

**Reply by sassanix** [reply]
Thank you so much, so if a model has vision capability it will be able to use it! I like this, but I wish I could also use it as a search engine to connect to ddg for example.

**Comment by tbendien** [comment]
This is a ridiculously cool capability of OWUI!!

**Comment by robheffo79** [comment]
Is there a way for the system to crawl the entire site and add it to the index?

**Reply by evolu8** [reply]
This would be amazing. Maybe could we restrict the search engine to the domain of a site?

**Comment by pedrobrantes** [comment]


**Reply by justinh-rahb** [reply]
Miss it? It didn't go anywhere. And if anything it's been replaced by something far superior: web search engine API support.

**Comment by fsdnotavai** [comment]
"[ERROR: 'NoneType' object has no attribute 'encode']"
